src/EnergyPlus/HVACControllers.cc

```diff
@@ -3555,6 +3520,47 @@ Label100: ;
 	}
 
 	void
+	GetControllerNameAndIndex(
```

Comment:
if controllers are to be simulated with water coils then water coils need to know which controller is attached.